model/fevm/trace.go

```diff
@@ -56,6 +56,10 @@ type FEVMTrace struct {
 	ParamsCodec uint64 `pg:",notnull,use_zero"`
 	// Returns codec.
 	ReturnsCodec uint64 `pg:",notnull,use_zero"`
+	// Human-readable identifier of receiver (To).
+	ToActorCode string `pg:",notnull"`
+	// Human-readable identifier of sender (From).
+	FromActorCode string `pg:",notnull"`
```

Comment:
We should call them `ToActorName` and `FromActorName`.

Reply:
sounds good, we should probably do it in other tables too then where `Code` is actually human readable name, [such as in this table in the schema](https://github.com/filecoin-project/lily/blob/master/schemas/v1/base.go#L118), to be consistent. How should we reconcile this? Should we rename the `actors` table to use `ActorName` too?

Reply:
Yes, we should. Btw, should be call it `actorName` or `actorType`?

Reply:
I think `ActorName` is good